core/Cargo.toml

```diff
@@ -15,6 +15,7 @@ codecov = { repository = "solana-labs/solana", branch = "master", service = "git
 
 [features]
 pin_gpu_memory = []
+move = []
```

Comment:
This feature doesn't seem to actually be used.

Reply:
```
error: failed to select a version for `solana-core`.
    ... required by package `solana-bench-tps v0.20.0 (/home/sakridge/src/solana/bench-tps)`
versions that meet the requirements `= 0.20.0` are: 0.20.0

the package `solana-bench-tps` depends on `solana-core`, with features: `move` but `solana-core` does not have these features.
```

Reply:
Remove `"solana-core/move"` at `bench-tps/Cargo.toml` Line 38